tests/karate/src/test/java/be/createRollCall/closeRollCall.feature

```diff
@@ -0,0 +1,29 @@
+@env=go,scala
+Feature: Close a Roll Call
+  Background:
+        # This is feature will be called to test Roll Call close
+        # The following calls makes this feature, mockFrontEnd.feature and server.feature share the same scope
+    * call read('classpath:be/utils/server.feature')
+    * call read('classpath:be/mockFrontEnd.feature')
+
+  Scenario: Close a valid roll should succeed
```

Comment:
String are usually in camelCase and here in snake_case. It would probably be good if we have a convention.
I see json are always in snake_case, is it intentional ?

There's also a typo in every rol_call_close at the start of every roll call close scenario